src/screens/ContactScreen/elements/ItemUser.js

```diff
@@ -1,13 +1,15 @@
 import * as React from 'react';
+import PropTypes from 'prop-types';
 import {Animated, Image, Pressable, StyleSheet, Text, View} from 'react-native';
 
-import MemoIc_Checklist from '../../../assets/icons/Ic_Checklist';
+import IconChecklist from '../../../assets/icons/Ic_Checklist';
 import {DEFAULT_PROFILE_PIC_PATH} from '../../../utils/constants';
 import {COLORS} from '../../../utils/theme';
 
-const ItemUser = ({photo, username, followed, onPress, userid}) => {
+const ItemUser = ({photo, username, followed, onPress, userid, isAnon}) => {
   const [isSelect, setIsSelect] = React.useState(false);
   const followIconFadeAnimation = React.useRef(new Animated.Value(0)).current;
+  const anonColorCondition = isAnon ? COLORS.anon_primary_15 : COLORS.signed_primary_15;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `anonColorCondition` variable is defined outside of the component's effect or render method. This could lead to unexpected behavior if the `isAnon` prop changes after the component has been initialized. Consider moving this inside the component function body to ensure it always reflects the current state of props.
```diff
- const anonColorCondition = isAnon ? COLORS.anon_primary_15 : COLORS.signed_primary_15;
  
  React.useEffect(() => {
    if (followed.includes(userid)) {
+   const anonColorCondition = isAnon ? COLORS.anon_primary_15 : COLORS.signed_primary_15;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->